popup.html

```diff
@@ -9,11 +9,40 @@
     		min-width:800px;
 				overflow: hidden;
 		}
+
+		#logo{
+			position: relative;
+			top: -60px;
+			left: 5px;
+			/* top: -60px; */
+			width: 128px;
+			height: 128px;
+			transform: scale(0.9);
+			margin: 0;
+			z-index: -1;
+		}
+
+		#Heading{
+			position: absolute;
+			/* display: flex;
+			flex-direction: column;
+			justify-content: center;
+			align-items: center; */
+			top: 15px;
+			width: 90%;
+			right: 20px;
+		}
+
+		
 	</style>
 	<body>
-
-			<h3 style="text-align: center;">Enter your ERP credentials here</h3>
-			<h5 style="text-align: center;">Saves As You Type</h5>
+		<header>
+			<div id="logo"><img src="logo256.png"></div>
+			<div id="Heading">
+				<h3 style="text-align: center;">Enter your ERP credentials here</h3>
+				<h5 style="text-align: center;">Saves As You Type</h5>
+			</div>
```

Comment:
I was just wondering, now that we have a logo on the top-left side of the window, it might look good to shift the header and other text at the top to the right by a small amount. What do you think?

Reply:
Thanks, @thealphadollar and Yeah sure I will try that.

Reply:
On the right, these setup looks better to me. Do comment if they are fine and if so which one looks better of the two.
![cmm2](https://user-images.githubusercontent.com/55590289/97086881-b29a2b00-1643-11eb-9e33-e5d5762d756b.jpg)
![cmm1](https://user-images.githubusercontent.com/55590289/97086883-b4fc8500-1643-11eb-9aa3-39253f336282.jpg)

Reply:
The former one looks better with the bold, in my opinion.